R/loo_compare.R

```diff
@@ -116,7 +122,34 @@ loo_compare.default <- function(x, ...) {
   diffs <- mapply(FUN = elpd_diffs, loos[ord[1]], loos[ord])
   elpd_diff <- apply(diffs, 2, sum)
   se_diff <- apply(diffs, 2, se_elpd_diff)
-  comp <- cbind(elpd_diff = elpd_diff, se_diff = se_diff, comp)
+
+  # compute probabilities that a model has worse elpd than the best model
+  # using a normal approximation (Sivula et al., 2025)
+  p_worse <- stats::pnorm(0, elpd_diff, se_diff)
+  p_worse[elpd_diff == 0] <- NA
+
+  # diagnostics to assess whether the normal approximation can be trusted
+  N <- nrow(diffs)
+  if (N < 100) {
+    # small N (Sivula et al., 2025)
+    diag_pnorm <- rep("N < 100", length(elpd_diff))
+    diag_pnorm[elpd_diff == 0] <- ""
+  } else {
+    diag_pnorm <- rep("", length(elpd_diff))
+    # similar predictions (Sivula et al., 2025)
+    diag_pnorm[elpd_diff > -4 & elpd_diff != 0] <- "similar predictions"
```

Comment:
Should this be `|elpd_diff| < 4` instead of "similar predictions" for consistency with the others? Or you could change the others to "small data" and "possible outliers". But right now there's a mix.